documentation/slate/source/includes/apis.md

```diff
@@ -972,7 +972,7 @@ It will retrieve all the journeys from the resource (in order to make *[isochron
 The [isochrones](#isochrones) service exposes another response structure, which is simpler, for the same data.
 
 ### <a name="journeys-disruptions"></a> Disruptions
-By default, Navitia only computes journeys that don't have disruptions, disrupted journeys will not be reported into the response.
+By default, Navitia only computes journeys without their associated disruption(s), meaning that the journeys in the response will be based on the theoritical schedules.
```

Comment:
Add the fact that the disruptions are present even without 'data_freshness=realtime' for information.